src/shared/components/entry/styles/entry-download.module.scss

```diff
@@ -0,0 +1,3 @@
+.select-group-label {
+  font-style: italic;
+}
```

Comment:
I don't think this is actually applying to anything at the moment, the select dropdown is not showing anything italicised within it for me (nor do I think this is supported by any browser)

Reply:
Yes, it was supposed to be removed. I will remove